src/data/Endpoint.js

```diff
@@ -51,6 +51,7 @@ const endpoints = {
     REQUEST_PASSWORD_RESET: `${laravelUrlPrefix}/${version}/user/password/email`,
     VALIDATE_PASSWORD_RESET_TOKEN: `${laravelUrlPrefix}/${version}/user/password/{token}`,
     LATEST_BACKGROUND_JOB: `${laravelUrlPrefix}/${version}/customer/background-job/latest/{0}`,
```

Comment:
I think there's an old country endpoint URL you want to remove @julianideamotive

Reply:
@maartenhemmes do you mean this one : ```COUNTRIES: `${urlPrefix}/countries`,``` ?

Reply:
@maartenhemmes old endpoint removed